src/v3/src/transformer/layout/email/emailChallengeConsentTransformer.ts

```diff
@@ -33,48 +31,33 @@ export const transformEmailChallengeConsent: IdxStepTransformer = ({ transaction
     } = {},
   } = transaction;
 
-  // removing consent field as it is controlled by buttons
-  uischema.elements = removeUIElementWithName(
-    'consent',
-    uischema.elements as UISchemaElement[],
-  );
-
   // @ts-ignore OKTA-489560 (missing requestInfo prop)
   const appName = requestInfo?.find((info) => info?.name === 'appName');
+  const appImageElement: ImageWithTextElement = {
+    type: 'ImageWithText',
+    options: {
+      id: appName.name,
+      SVGIcon: AppSvg,
+      textContent: appName.value,
+    },
+  };
   // @ts-ignore OKTA-489560 (missing requestInfo prop)
   const browser = requestInfo?.find((info) => info?.name === 'browser');
-
-  if (appName?.value) {
-    const appImageElement: ImageWithTextElement = {
-      type: 'ImageWithText',
-      options: {
-        id: appName.name,
-        SVGIcon: AppSvg,
-        textContent: appName.value,
-      },
-    };
-    uischema.elements.unshift(appImageElement);
-  }
-
-  if (browser?.value) {
-    const browserImageElement: ImageWithTextElement = {
-      type: 'ImageWithText',
-      options: {
-        id: browser.name,
-        SVGIcon: BrowserSvg,
-        textContent: browser.value,
-      },
-    };
-    uischema.elements.unshift(browserImageElement);
-  }
+  const browserImageElement: ImageWithTextElement = {
+    type: 'ImageWithText',
+    options: {
+      id: browser.name,
+      SVGIcon: BrowserSvg,
+      textContent: browser.value,
```

Comment:
Is this possibly undefined? TS doesnt complain about this?

Reply:
there is guard when reassign uischema.elements

Reply:
or you mean it should guard against browser instead of browser.value?